gcc/config/riscv/sifive-vector.md

```diff
@@ -182,3 +182,599 @@
   "sf.vfnrclip.x<v_su>.f.qf\t%0,%3,%4%p1"
   [(set_attr "type" "sf_vfnrclip")
    (set_attr "mode" "<MODE>")])
+
+;; SF_VCP
+(define_insn "@sf_vc_x<mode>"
+  [(set (match_operand:VFULLI 0 "register_operand" "=vd")
+	(if_then_else:VFULLI
+	  (unspec:<VM>
+	    [(match_operand:<VM> 1 "vector_mask_operand"             "  Wc1")
+	     (match_operand 7 "vector_length_operand"                "   rK")
+	     (match_operand 8 "const_int_operand"                    "    i")
+	     (match_operand 9 "const_int_operand"                    "    i")
+	     (match_operand 10 "const_int_operand"                   "    i")
+	     (reg:SI VL_REGNUM)
+	     (reg:SI VTYPE_REGNUM)] UNSPEC_VPREDICATE)
+	  (unspec:VFULLI
+	    [(match_operand:SI 3 "const_int_operand" "B")
+	     (match_operand:SI 4 "const_int_operand" "K")
+	     (match_operand:SI 5 "const_int_operand" "K")
+	     (match_operand:<VEL> 6 "register_operand" "r")] UNSPEC_SF_CV)
+       (match_operand:VFULLI 2 "vector_merge_operand"     "vu")))]
+  "TARGET_VECTOR && TARGET_XSFVCP"
+  "sf.vc.x\t %3, %4, %5, %6"
+  [(set_attr "type" "sf_vc_se")
+   (set_attr "mode" "<MODE>")])
```

Comment:
For those instruction without output, you don't really need a dummy output operand there, you just need some thing like